build.py

```diff
@@ -312,6 +313,7 @@ def build_package(target):
         i = re.sub("TEMPLATE_CLANG_CPP_LIBRARIES", clang_cpp_libraries, i)
         i = re.sub("TEMPLATE_CLANG_SUBDIRECTORY", clang_subdirectory, i)
         i = re.sub("TEMPLATE_CLANG_EXECUTABLE", clang_executable, i)
+        i = re.sub("TEMPLATE_CLANG_C_EXECUTABLE", clang_c_executable, i)
```

Comment:
i think this is the same as `TEMPLATE_CLANG_EXECUTABLE`

Reply:
done